<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Accept</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="OpenFileBtn">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>打开文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FileLineEdit">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>文件名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SendFileBtn">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>发送文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReceiveStopBtn">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReceiveSaveBtn">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>保存窗口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReceiveClearBtn">
           <property name="minimumSize">
            <size>
             <width>71</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>清除窗口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="HelpBtn">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>帮助</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DownControlBtn">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExpandBtn">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>扩展</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>数据位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dataBitsComboBox">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>串口号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="portNameComboBox">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="baudRateComboBox">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>流控</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>Hardware</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Software</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>停止位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="stopBitsComboBox">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>校验位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="parityComboBox">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>20</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Even</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Odd</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QPushButton" name="openMyComBtn">
                    <property name="text">
                     <string>打开串口</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                     <widget class="QCheckBox" name="HEXDisplay">
                      <property name="text">
                       <string>HEX显示</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="HEXSend">
                      <property name="text">
                       <string>HEX发送</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QCheckBox" name="AutoSend">
                    <property name="text">
                     <string>定时发送</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="AutoSendTime"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>ms</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>字符输入框：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="sendMsgBtn">
                    <property name="text">
                     <string>发送</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>欢迎使用串口调试助手
版本号UartAssistant_v0
作者：w</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="sendMsgLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
